### PR TITLE
Envelope uses ES6 class

### PR DESCRIPTION
**Pull request by Ajaya1000**
changed to es6 class structure

### CODE REVIEW

File: src/envelope.js [hunk 1]
```diff
@@ -48,852 +48,858 @@ import TimelineSignal from 'Tone/signal/TimelineSignal.js';
...
-  for (var i = 1; i < this.mathOps.length; i++) {
-    this.mathOps[i].dispose();
+}
+export class Env extends Envelope {
+  constructor(t1, l1, t2, l2, t3, l3) {
+    console.warn(
+      'WARNING: p5.Env is now deprecated and may be removed in future versions. ' +
+        'Please use the new p5.Envelope instead.'
+    );
+    super();
```
Comment (endurance21) [inline src/envelope.js:902]: pass the arguements of constructor to super() too.
We have a different aproach to handle the deprecations now.
please put this ENV to src/deprecations .

**Review by endurance21 — CHANGES_REQUESTED** [review]

**Comment by endurance21** [comment]
@Ajaya1000  also check the examples if the p5.envelope is runnig fine https://github.com/processing/p5.js-sound/blob/main/examples/envelope/sketch.js
and let me know if everything is fine .
I wil verify and then we can merge this one

**Comment by Ajaya1000** [comment]
@endurance21 I have made the suggested changes and all examples using envelope classes are working as expected.

**Review by endurance21 — APPROVED** [review]

**Comment by Abhijay007** [comment]
Hi @Ajaya1000  can you please look into the merge conflicts, if you are busy I can resolve those conflicts and make a new PR  fixing merge conflicts around the same :)

**Comment by davepagurek** [comment]
I'm going to close this for organizational purposes since the changes have been merged in as part of @Abhijay007's PR. Thanks for your changes @Ajaya1000!

**Pull request closed without merging**